packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -167,17 +162,54 @@ async function initialize(
   projectSourceRoot?: string;
   i18n: I18nOptions;
 }> {
+  if (!context.target) {
+    throw new Error('The builder requires a target.');
+  }
+
+  const metadata = await context.getProjectMetadata(context.target);
+  const i18n = createI18nOptions(metadata, options.localize);
+
+  if (i18n.inlineLocales.size > 0) {
+    // Load locales
+    const loader = await createTranslationLoader();
+
+    const usedFormats = new Set<string>();
+    for (const [locale, desc] of Object.entries(i18n.locales)) {
+      if (i18n.inlineLocales.has(locale)) {
+        const result = loader(desc.file);
+
+        usedFormats.add(result.format);
+        if (usedFormats.size > 1) {
+          // This limitation is technically only for legacy message id support
+          throw new Error(
+            'Localization currently only supports using one type of translation file format for the entire application.',
+          );
+        }
+
+        desc.format = result.format;
+        desc.translation = result.translation;
+      }
+    }
+
+    // Legacy message id's require the format of the translations
+    if (usedFormats.size > 0) {
+      options.i18nFormat = [...usedFormats][0];
+    }
+  }
+
+  const originalOutputPath = options.outputPath;
+
+  // If inlining store the output in a temporary location to facilitate post-processing
+  if (i18n.shouldInline) {
+    options.outputPath = fs.mkdtempSync(path.join(fs.realpathSync(os.tmpdir()), 'angular-cli-'));
```

Comment:
As far as I recall, on Windows temp directories are not cleaned automatically, this means that for every build we will create a temporary directly. 

Why not use `find-cache-dir`

Reply:
Added code to delete the temporary directory on completion of the i18n processing.

I think a temporary directory is more appropriate since the output is transient. System temp locations are generally optimized by the operating system (automatically cleaned or potentially mounted as a memory filesystem).  This will also eventually be replaced with a build setup that will keep the files in memory throughout the pipeline and eliminate the need for this.